### ISSUE TITLE
[Gallery][Skia.Gtk]when scrolling up, all the option(Material, Fluent, Cupertino) moves to the left side and these options gets invisible.

### ISSUE THREAD

**Issue by sakshi173** [unoplatform/uno#11659]
### Current behavior

https://user-images.githubusercontent.com/74563602/224601803-3d8f27ce-fc0b-4f05-9c08-c5a146b93ffb.mp4

![button issue](https://user-images.githubusercontent.com/74563602/224601839-8340ee1d-e80a-4e85-99f6-a1c167e00c96.png)
![scrolling issue](https://user-images.githubusercontent.com/74563602/224601856-54c038b6-af92-4a3d-9da3-44aa92d3ee13.png)

### Expected behavior

![button correct](https://user-images.githubusercontent.com/74563602/224602336-a146215a-fa63-49ae-84b7-910ee0fd4576.png)

### How to reproduce it (as minimally and precisely as possible)

1. Open Uno Gallery APP
2. Tap Burger Menu on the left top right.
3. Tap on 'Button' or any option from the "UI Component" list.
4. Use mouse cursor and scroll up.
5. Notice that all the option(Material, Fluent, Cupertino) moves to the left side and these options gets invisible.

### Works on UWP/WinUI

None

### Affected platforms

Skia (GTK on Linux/macOS/Windows)

**Comment by sakshi173** [comment]
This issue is still present, please look into it.

**Comment by nguyeenvy** [comment]
@Marc-Antoine-Soucy Wasn't there a PR that fixed this issue at some point this summer?

**Comment by jeromelaban** [comment]
@sakshi173 is this issue still reproducing?

**Comment by sakshi173** [comment]
@jeromelaban, @nguyeenvy , yes this issue is still present. below is the screenshot from the latest build.
![scrolling issue](https://github.com/unoplatform/uno/assets/74563602/33f464f2-b147-492f-9bf0-937c73714d5e)

**Comment by sakshi173** [comment]
Verified this issue is fixed, closing it.
